public/js/mage/adminhtml/tools.js

```diff
@@ -701,7 +701,21 @@ var Base64 = {
             }
         }
         return string;
-    }
+    },
+
+    toArrayBuffer: function(base64) {
+        const binaryString = atob(base64);
```

Comment:
```suggestion
        const binaryString = this.mageDecode(base64);
```

Because the base64 from the controller is in Magento's special format.

Or, I think we could not replace the chars here:
```php
            $challenge = strtr(base64_encode(random_bytes(32)), '+/', '-_');
```